doc/manual/src/command-ref/nix-env/delete-generations.md

```diff
@@ -9,14 +9,36 @@
 # Description
 
 This operation deletes the specified generations of the current profile.
-The generations can be a list of generation numbers, the special value
-`old` to delete all non-current generations, a value such as `30d` to
-delete all generations older than the specified number of days (except
-for the generation that was active at that point in time), or a value
-such as `+5` to keep the last `5` generations ignoring any newer than
-current, e.g., if `30` is the current generation `+5` will delete
-generation `25` and all older generations. Periodically deleting old
-generations is important to make garbage collection effective.
+
+*generations* can be a one of the following:
+
+- `<number>...`: A list of generation numbers, each one a separate command-line argument.
+
+  Delete exactly the profile generations given by their generation number.
+  Deleting the current generation is not allowed.
+
+- The special value `old`
+
+  Delete all generations older than the current one.
+
+- `<days>d`: The last *days* days
+
+  *Example*: `30d`
+
+  Delete all generations older than *days* days.
+  The generation that was active at that point in time is excluded, and will not be deleted.
+
+- `+<count>` The last *count* generations up to the present
+
+  *Example*: `+5`
+
+  Keep the last *count* generations, along with any newer than current.
+
+Periodically deleting old generations is important to make garbage collection
+effective.
+The is because profiles are also garbage collection roots --- any [store object] reachable from a profile is "alive" and ineligible for deletion.
```

Comment:
```suggestion
The is because profiles are also garbage collection roots – any [store object] reachable from a profile is "alive" and ineligible for deletion.
```

Reply:
```suggestion
The is because profiles are also garbage collection roots — any [store object] reachable from a profile is "alive" and ineligible for deletion.
```